CAIPs/caip-25.md

```diff
@@ -16,422 +16,247 @@ CAIP-25 defines an authorization procedure for a chain agnostic provider to inte
-      "eip155:42161": {
-        "extension_foo": "bar"
+      "solana:5eykt4UsFv8P8NJdTREpY1vzqKqZKvdp": {
+        "methods": [
+          "solana_signMessage",
+          "solana_signTransaction",
+          "solana_signAndSendTransaction"
+        ],
+        "notifications": [],
+        "capabilities": {
+          "supportedTransactionVersions": ["legacy", "0"]
+        }
+      },
+      "solana:4uhcVJyU9pJkvQyS88uRDiswHXSCkY3z": {
+        "methods": [
+          "solana_signMessage",
+          "solana_signTransaction",
+          "solana_signAndSendTransaction"
+        ],
+        "notifications": [],
+        "capabilities": {
+          "supportedTransactionVersions": ["legacy"]
+        }
       }
     },
-    "sessionProperties": {
+    "properties": {
       "expiry": "2022-12-24T17:07:31+00:00",
-      "caip154-mandatory": "true"
+      "caip369-personalInfo": "true"
     }
   }
 }
 ```
 
-The JSON-RPC method is labeled as `wallet_createSession` and its `params` object contains "requiredScopes" and/or "optionalScopes" objects populated with [CAIP-217][] "scope objects" keyed to [CAIP-217][] scope strings.
-
-- The `requiredScopes` object MUST contain 1 or more `scopeObjects`, if present.
-- The `optionalScopes` object MUST contain 1 or more `scopeObjects`, if present.
-
-A third object is the `scopedProperties` object, which also MUST contain 1 or more objects if present.
-Each object should be keyed to the scope of a `sessionScopes` member to which it corresponds.
-All properties of each object in `scopedProperties` MUST be interpreted by the respondent as proposals or declarations rather than as requirements.
-In addition to making additional properties of or metadata about the corresponding `sessionScopes` member explicit, they can also annotate, support, or extend the negotiation of scope proposals (e.g., providing connection information about unfamiliar scopes, or which accounts to expose to each).
+The `scopes` object MUST contain one or more scopeObjects.
 
-A fourth object, `sessionProperties`, is optional and its shape undefined.
-It is intended for metadata or additional information not bound to any specific authorization scope, but made "global" to the connection.
-
-The respondent SHOULD ignore and drop from its response any properties not defined in this document or in another CAIP document extending this protocol which the respondent has implemented in its entirety;
-similarly, the `requiredScopes`, `optionalScopes`, and `sessionScopes` objects returned by the respondent SHOULD contain only valid [CAIP-217][] objects, and properties not defined in [CAIP-217][] SHOULD also be dropped from each of those objects.
-The same absolute security posture is not expected for the metadata objects `scopedProperties` and `sessionProperties`, but caution is still recommended for such extensions:
-callers and respondents alike SHOULD allow for their counterparties dropping or ignoring unfamiliar members from either.
```

Comment:
i feel like some of this guidance is helpful. Is it worth giving up these sections to make the spec appear simpler?

Reply:
Move them to non-normative appendices, maybe?